i18n.config.ts

```diff
@@ -116,8 +117,9 @@ export default defineI18nConfig(() => ({
     uk,
     ug,
     vi,
-    'zh-CN': zhCn,
-    'zh-HK': zhHk,
-    'zh-TW': zhTw,
-  }
-}))
+    "zh-CN": zhCn,
+    "zh-HK": zhHk,
+    "zh-TW": zhTw,
+    sv,
```

Comment:
Can you put `sv` in alphabetical order below `sk`?

Reply:
hello!, i have updated it